Login UseCase/AuthenticationServiceTest.java

```diff
@@ -1,10 +1,9 @@
 import org.junit.Before;
 import org.junit.Test;
-
 import static org.junit.Assert.*;
 
```

Comment:
You Can Add tests for edge cases, such as duplicate usernames or invalid input formats.